Milestone3m [WiP] (#369)
linkeddata/mashlib: README.md

A major limitation of this data browser hack is that current web browsers are made to distrust any code loaded from one domain that uses data from another domain. This makes it hard, strangely complicated, and sometimes impossible to do some things.

## Generative AI usage
The SolidOS team is using GitHub Copilot integrated in Visual Studio Code. 
We have added comments in the code to make it explicit which parts are 100% written by AI. 

### Prompt usage history:

> * Auto model: Looking at these 2 files (databrowser.html and index.ts), I want to redesign mashlib and underlying panes. I have a design for web and one for mobile. How would I go about making sure I can also have a mobile version? 

Reviewer: ```suggestion
* Auto model: Looking at these 2 files (`databrowser.html` and `index.ts`), I want to redesign mashlib and underlying panes. I have one design for web and another for mobile. How would I go about making sure I can also have a mobile version? 
```

> * Claude-Opes 4.6: I don't think this is correct. Mashlib is bundling together all the panes. I do not need to add mashlib.layout or theme to the globals. I can just call the render of each pane with an interface of values or?

Reviewer: ```suggestion
* Claude-Opes 4.6: I don't think this is correct. Mashlib is bundling together all the panes. I do not need to add `mashlib.layout` or theme to the globals. I can just call the render of each pane with an interface of values or ...?
```

> * Raptor mini: I want to rdesign the main page which contains GlobalDashboard and OtlineView. I keep the header and footer and I want to have a left side menu that chnages content base on logged in or not and also chnages if it is mobile or not. If it is mobile it should fold up in the header menu if it is web it should be rendered on the left side. I want a menu placeholder that changes according to mobile or not. Its functionality I want to implement in solid-panes.

Reviewer: ```suggestion
* Raptor mini: I want to redesign the main page which contains GlobalDashboard and OutlineView. I keep the header and footer and I want to have a left side menu that changes content based on whether logged in or not and also changes based on whether it is mobile or not. If it is mobile, it should fold up in the header menu; if it is web, it should be rendered on the left side. I want a menu placeholder that changes according to mobile or not. I want to implement its functionality in solid-panes.
```
